playbook/app/pb_kits/playbook/pb_icon/_icon.jsx

```diff
@@ -12,7 +12,7 @@ type IconProps = {
   data?: object,
   fixedWidth?: boolean,
   flip?: "horizontal" | "vertical" | "both" | "none",
-  icon: string,
+  icon: string | any,
```

Comment:
We only want this in the case that we might want to allow an actual SVG to be passed here instead of just a string